forge/db/controllers/Project.js

```diff
@@ -11,6 +11,27 @@ const inflightProjectState = { }
 
 const inflightDeploys = new Set()
 
+class ControllerError extends Error {
```

Comment:
The long term intension would be to have a standard set of FlowForge error objects that we use throughout the app at various logical levels.